src/pages/wishlist.html

```diff
@@ -0,0 +1,200 @@
+							<div class="card-badge">ON SALE</div>
+							<span><i class="fas fa-heart"></i></span>
+							<span class="hidden"><i class="fal fa-times-circle"></i></span>
+						</div>
+						<div class="card-overlay-text-div hidden">
+							<div class="overlay-text">Out of stock</div>
+						</div>
+						<div class="card-img-div">
+							<img src="../assets/images/product_listing/product_1.jpg" class="card-img" alt="" />
+						</div>
+					</div>
+					<div class="card-footer">
+						<div class="card-text">Endurance Way-2 Headphones</div>
+						<div class="card-pricing"><strike>Rs. 1499</strike> &nbsp; Rx. 1199</div>
+						<button class="card-action-btn"><i class="far fa-shopping-cart"></i> &nbsp; ADD TO CART</button>
+					</div>
+				</div>
+
+				<div class="card card-v floating-shadow">
+					<div class="card-body flex justify-center">
+						<div class="card-header flex justify-between">
+							<div class="card-badge">ON SALE</div>
+							<span><i class="fas fa-heart"></i></span>
+							<span class="hidden"><i class="fal fa-times-circle"></i></span>
+						</div>
+						<div class="card-overlay-text-div hidden">
+							<div class="overlay-text">Out of stock</div>
+						</div>
+						<div class="card-img-div">
+							<img src="../assets/images/product_listing/product_1.jpg" class="card-img" alt="" />
+						</div>
+					</div>
+					<div class="card-footer">
+						<div class="card-text">Endurance Way-2 Headphones</div>
+						<div class="card-pricing"><strike>Rs. 1499</strike> &nbsp; Rx. 1199</div>
+						<button class="card-action-btn"><i class="far fa-shopping-cart"></i> &nbsp; ADD TO CART</button>
+					</div>
+				</div>
+
+				<div class="card card-v floating-shadow">
+					<div class="card-body flex justify-center">
+						<div class="card-header flex justify-between">
+							<div class="card-badge">ON SALE</div>
+							<span><i class="fas fa-heart"></i></span>
+							<span class="hidden"><i class="fal fa-times-circle"></i></span>
+						</div>
+						<div class="card-overlay-text-div hidden">
+							<div class="overlay-text">Out of stock</div>
+						</div>
+						<div class="card-img-div">
+							<img src="../assets/images/product_listing/product_1.jpg" class="card-img" alt="" />
+						</div>
+					</div>
+					<div class="card-footer">
+						<div class="card-text">Endurance Way-2 Headphones</div>
+						<div class="card-pricing"><strike>Rs. 1499</strike> &nbsp; Rx. 1199</div>
+						<button class="card-action-btn"><i class="far fa-shopping-cart"></i> &nbsp; ADD TO CART</button>
+					</div>
+				</div>
+
+				<div class="card card-v floating-shadow">
+					<div class="card-body flex justify-center">
+						<div class="card-header flex justify-between">
+							<div class="card-badge">ON SALE</div>
+							<span><i class="fas fa-heart"></i></span>
+							<span class="hidden"><i class="fal fa-times-circle"></i></span>
+						</div>
+						<div class="card-overlay-text-div hidden">
```

Comment:
Will suggest not to use the word 'div' in class names. Think of it as a reserved word (it's not actually)

Reply:
Sure Shashi, Will keep in mind. thanks.